src/vnsw/agent/contrail/windows/msi/agent_msi.wxs

```diff
@@ -24,7 +26,14 @@
                         </Component>
                         <Component Id="AgentServiceWrapperExe" Guid="8A6640CF-5EA0-4F16-A006-0991147E0D9E" Win64="yes">
                             <File Id="AgentServiceWrapperExe" Name="contrail-vrouter-agent-sw.exe" Source="build/bin/winsw.exe"/>
-                            <ServiceInstall Id="CreateService" Name="ContrailAgent" DisplayName="ContrailAgent" Type="ownProcess" Start="demand" ErrorControl="normal"/>
+                            <ServiceInstall Id="CreateService" Name="ContrailAgent" DisplayName="ContrailAgent" Type="ownProcess" Start="demand" ErrorControl="normal">
+                                <util:ServiceConfig
```

Comment:
Why didn't you use ServiceConfigFailureActions ? I think it doesn't require util extension.